io/js/src/main/scala/fs2/io/net/tls/TLSSocketPlatform.scala

```diff
@@ -37,6 +37,18 @@ private[tls] trait TLSSocketPlatform[F[_]]
 private[tls] trait TLSSocketCompanionPlatform { self: TLSSocket.type =>
 
   private[tls] def forAsync[F[_]](
```

Comment:
Should we rename this `forAsyncServer`?

Reply:
Or couldn't we just pass `clientMode` in `forAsync` so we don't need separate methods? I don't really care, these are just nitpicks :)

Reply:
Oh I see, it was bincompat. But this is all private anyway. We can just add the exclusion.

Reply:
> Since it was fairly easy to avoid I did

Yeah, this is good practice so thanks for doing that. But in this case I think its cleaner like this.